Katello/katello
# Fixes #36687 - Changed button text to 'Save' when editing content vie…

…w errata by date range filters and rpm filters

#### What are the changes introduced in this pull request?
The confirm button when editing content view errata by date range filters and content view rpm filters now says 'Save' instead of 'Edit filter'. 

#### Considerations taken when implementing this change?
n/a

#### What are the testing steps for this pull request?

1. Create a content view with an RPM filter and an Errata - by date range filter.
2. Enter the RPM filter and add a rule. Edit that rule. The confirm button in the lower left of the modal should read 'Save'.
3. Enter the Errata - by date range filter. The confirm button in the lower left of the page should read 'Save'.
